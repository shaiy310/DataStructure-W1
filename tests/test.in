Init
AddStudent 1 0 2
AddStudent 1 2 2
AddStudent 2 1 4
AddTeam 1
MoveStudentToTeam 1 1
GetMostPowerful 2
GetMostPowerful 1
MoveStudentToTeam 2 1
GetMostPowerful 1
IncreaseLevel 0 3
GetAllStudentsByPower 1
Quit