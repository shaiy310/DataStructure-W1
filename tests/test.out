Init done.
AddStudent: SUCCESS
AddStudent: FAILURE
AddStudent: SUCCESS
AddTeam: SUCCESS
MoveStudentToTeam: SUCCESS
GetMostPowerful: FAILURE
Most powerful student is: 1
MoveStudentToTeam: SUCCESS
Most powerful student is: 2
IncreaseLevel: SUCCESS
Rank	||	Student
1		||	1
2		||	2
and there are no more students!
Quit done.
